packages/remix-cloudflare-workers-esm/tsconfig.json

```diff
@@ -0,0 +1,22 @@
+{
+  "include": ["**/*.ts"],
+  "exclude": ["dist", "__tests__", "node_modules"],
+  "compilerOptions": {
+    "lib": ["ES2019"],
+    "target": "ES2019",
+    "types": ["@types/node", "@cloudflare/workers-types"],
```

Comment:
Perhaps controversial, but I moved the `/// <reference />` out of `worker.ts` and into `tsconfig.json`. This is [best practice according to Microsoft](https://www.typescriptlang.org/docs/handbook/triple-slash-directives.html#-reference-types-).

In an ideal world we’d probably just do:

```ts
declare process {
  env: {
    NODE_ENV?: string;
  }
}
```